### ISSUE TITLE
Is there an example of updating the camera initial position?

### ISSUE THREAD

**Issue by cysidron1** [gabotechs/react-stl-viewer#41]
Hello there, I'm enjoying using this package in a project I'm working on - however, I am wondering if you have any guidance on using the CameraInitialPosition interface? Since the StlViewer component does not accept that in its props, it's not clear to me how I can update the values for the initial camera position and have my viewer use those.

**Comment by gabotechs** [comment]
Right now, even though the `CameraInitialPosition` interface is exposed, there's no way of providing it to the `StlViewer` component, it's just used internally.

It would be a nice addition to expose it and let users set their own `CameraInitialPosition`. I will make a PR and get that working between today and tomorrow.

**Comment by gabotechs** [comment]
passing the `cameraInitialPosition` is already available in the `2.4.0` release. You would just pass it to the `StlViewer` component as any other prop:
```tsx
<StlViewer
  cameraInitialPosition={{
    latitude: Math.PI / 8,
    longitude: -Math.PI / 8,
    distance: 2.5
  }}
>
```
I've also updated the README describing how this parameter works, if you see that I've missed something feel free to open a new issue.
